releasenotes/notes/clearing-sessions-now-only-removes-traces,-instead-of-sessions,-requsting-traces-will-now-return-a-404-error-for-TracesNotFound-if-a-session-is-specified-and-traces-are-not-found-6b0311673d8b37aa.yaml

```diff
@@ -0,0 +1,13 @@
+---
+features:
+  - |
+    Updates the logic for associating requsets with a session.  Prior to this change:
+    (1) non-existent tokens were permitted and returned associations with all requests, 
+    (2) existent tokens returned associations with all matching requests + all untokenized requests after the session was created
+
+    After this change:
+    (1) Requests with non-existent tokens return 400 error codes
+    (2) Requests with existent tokens return all matching requests + all untokenized requests after the session was created but 
```

Comment:
I think we still need a test for the latter part of (2)

Reply:
that is tested under test_session_association_of_untokenized_traces